src/config-loader/config-loader.coffee

```diff
@@ -93,6 +94,7 @@ exports.setup = (app) ->
 				Promise.map data.models, (model) ->
 					if model.modelText?
 						apiRoute = '/' + model.apiRoot + '/*'
+						app.post('/' + model.apiRoot + '/([\$])canAccess', sbvrUtils.authorizationMiddleware, resourceaccess.canAccessV1Request)
```

Comment:
I think this should actually be handled by the already added `*` path below, the same way other meta endpoints like `$metadata` are, and hook into the same `handleODataRequest` chain, that way internal api requests will be able to be made to this endpoint rather than only being able to make external http calls

Reply:
Sounds good. will use the same chain as it is used for `$metadata`.